app/utils/cache/cache.go

```diff
@@ -0,0 +1,58 @@
+package cache
+
+import (
+	"sync"
+	"time"
+)
+
+type cacheMap struct {
+	mutex sync.Mutex
+	items map[string]cacheItem
+}
+
+type cacheItem struct {
+	expires time.Time
+	item    any
+}
+
+var appCacheMap cacheMap
+
+func init() {
+	appCacheMap.items = make(map[string]cacheItem)
+}
+
+// GetCachedItem returns cached item and true for provided cache key if item is fresh in cache.
+func GetCachedItem(key string) (any, bool) {
```

Comment:
Since we are already inside package `cache`, we don't need the "cached" anymore in the name as it's redundant.
So instead of `cache.GetCahcedItem`, I can just write: `cache.GetItem` or even shorted `cache.Get`.

Probably this interface would be most concise and universal:
```golang
cache.Get(key string) (any, bool)
cache.Set(key string, item any, ttl time.Duration)
cache.Delete(key string)
```